datarobot_batch_scoring/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.6.0a4'
+__version__ = '1.6.1a2'
```

Comment:
Since new API (functionality?) is introduced, Minor version should be bumped, not patch (in case if we use [semver](http://semver.org/)).